### PR TITLE
PAYEE-213 - Implemented UR banner

### PR DESCRIPTION
**Pull request by brendanstraughan**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

**Review by masahmad — APPROVED** [review]
LGTM

File: app/views/MainTemplate.scala.html [hunk 1]
```diff
@@ -192,7 +193,7 @@
 }
 
 @templateArguments = @{
-    if(!isUserResearchBannerHidden && appConfig.urBannerEnabled) {
+    if(!isUserResearchBannerHidden && (appConfig.urBannerEnabled || showUrBannerOverride)) {
```
Comment (sergi-hmrc) [inline app/views/MainTemplate.scala.html:196]: is it correct to ignore the appConfig if a message sets showUrBannerOverride to true?

**Review by sergi-hmrc — APPROVED** [review]

File: app/views/MainTemplate.scala.html (hunk 1 above)
Comment (brendanstraughan) [reply app/views/MainTemplate.scala.html:196]: yeah this is the functionality we want as we want to be able to show the banner on one page only

**Review by vivekhmrc — APPROVED** [review]

**Review by Jamie22Butler — APPROVED** [review]
LGTM 👍

File: conf/messages.cy [hunk 2]
```diff
@@ -1411,8 +1411,8 @@ tai.taxCodeComparison.employments.count=mae gennych {0} o gyflogaethau nawr
 tai.taxCodeComparison.pension.count=erbyn hyn mae gennych {0} pensiwn
 tai.taxCodeComparison.pensions.count=erbyn hyn mae gennych {0} bensiwn
 tai.taxCodeComparison.incomeSources.count=erbyn hyn mae gennych {0} ffynhonnell incwm
-tai.urbanner.title=Helpwch i wella llythyrau CThEM
-tai.urbanner.text=Cymerwch ran yn ein harolwg byr (yn agor ffenestr neu dab newydd)
+tai.urbanner.title=Helpwch i wella GOV.UK
+tai.urbanner.text=Helpwch i wella’r gwasanaeth TWE drwy gymryd rhan mewn arolwg (yn agor ffenestr newydd)
```
Comment (vivekhmrc) [inline conf/messages.cy:1415]: Looks good 👍

**Review by vivekhmrc — APPROVED** [review]

**Pull request merged**